src/automate.js

```diff
@@ -59,28 +61,46 @@ function drawAutomatedStackFrames(stack_frames, configuration) {
     // The width required for drawing stack-frames (corresponding to the maximum width among all the stack-frames)
     let required_width = 0;
 
+    // The stack frames that will be drawn (the stack frames that are not blank
+    let draw_stack_frames = [];
+
+
     // Loop through all the stack-frames to determine their individual box height
     for (const stack_frame of stack_frames){
 
+        // Initialize the width and the height of the stack frames
+        let width;
+        let height;
+
         // get the width and height of each box
-        const {height, width} = getSize(stack_frame)
+        if (stack_frame.name !== 'BLANK') {
+            height = getSize(stack_frame).height;
```

Comment:
Don't repeat function calls unnecessarily. `getSize` should only be called once here (as it was before)

Reply:
@umututku03 it looks like you resolved this comment (and others) without actually fixing them. That's not good.

Reply:
@david-yz-liu Hello Professor Liu, it looks like we did not commit (and push) those changes to the branch that we have created the pull request. I am sorry for the confusion. We definitely intended to actually resolve these comments. Resolving those without actually fixing them is something that is strictly against our work ethic.